### PR TITLE
Ap/more recursion

### PR DESCRIPTION
**Pull request by inactdev**
Please work !!

### CODE REVIEW

File: lib/grape/jsonapi/exposer.rb [hunk 1]
```diff
@@ -21,6 +21,16 @@ def self.field(instance, field)
         end
       end
 
+      def self.one_level_deep?(field)
+        field_exist = field_exists?(field)
+        lambda do |instance, options|
+          if field_exist.call(instance, options)
+            return true if options.opts_hash.dig(:attr_path).count(:included) == 1
```
Comment (alexshook) [inline lib/grape/jsonapi/exposer.rb]: I think you can get rid of the `return true if` here.

Comment (alexshook) [reply lib/grape/jsonapi/exposer.rb]: Nm I see that's an `end`, not an `else` below.

Comment (chandeeland) [reply lib/grape/jsonapi/exposer.rb]:
```
depth = options.opts_hash.dig(:attr_path).count(:included)
field_exist.call(instance, options) && depth == 1
```

**Review by chandeeland — CHANGES_REQUESTED** [review]
bump the version yo

**Review by chandeeland — APPROVED** [review]

**Pull request merged**